exla/guides/rotating-image.livemd

```diff
@@ -1,14 +1,13 @@
 # Rotating Image
 
-## Setup
-
 ```elixir
 Mix.install(
   [
-    {:req, "~> 0.2.0"},
-    {:kino, "~> 0.5.1"},
-    {:exla, "~> 0.2"},
-    {:stb_image, "~> 0.5"}
+    {:nx, github: "elixir-nx/nx", override: true, sparse: "nx"},
```

Comment:
```suggestion
```

EXLA should bring Nx, no?

Reply:
Yes, but it'll bring 0.5, but `Nx.Constants.pi` is on the main branch